### DISCUSSION TITLE
Best way to handle errors?

### DISCUSSION THREAD

**Discussion by ghost** [wailsapp/wails#2831] [Q&A]
Hello! I'm just wondering? How are y'all handling errors? Because the errors return a string. Is there a way to send like an error code so the frontend can know what type of error it is? Because using ```includes``` isn't really ideal I think.

Thanks.

**Comment by marcio199226** [comment]
You could return just those error codes from methods called directly from js runtime

**Comment by leaanthony** [comment]
There was a PR merged recently to allow custom error handling. Should be in next release.
